packages/helm-charts/grafana/values-clabs.yaml

```diff
@@ -11,6 +11,10 @@ datasources:
       name: Prometheus
       type: prometheus
       url: http://prometheus-server.prometheus:9090
+    - access: proxy
+      name: Loki
+      type: loki
+      url: http://local-loki:3100
```

Comment:
Is this already set up anywhere?
Wouldn't this additionally require to send the logs to it from promtail?

Reply:
@alvarof2 [confirmed](https://github.com/celo-org/infrastructure/issues/281#issuecomment-1136906720) me that Loki and an additional local promtail were deployed manually to Forno clusters when he reduced the log volume in grafana cloud (most of them were coming from kong/nginx logs in forno).
We can the local-loki datasource in local grafana just to forno clusters, but as it is something that does not hurt (the extra datasource) and loki has been deployed manually to some of the clusters, also I think it makes sense to have it added to all local grafana instances, and install loki using helm wherever it is needed.

Reply:
Can you add a comment about it since it's non obvious from reading the config?